Sources/Dependencies/DependencyValues/OpenURL.swift

```diff
@@ -12,7 +12,7 @@
 
   @available(iOS 14, macOS 11, tvOS 14, watchOS 7, *)
   private enum OpenURLKey: DependencyKey {
-    static let liveValue = OpenURLEffect { url in
+    static let liveValue = OpenURLEffect { @MainActor url in
```

Comment:
```suggestion
    // NB: This must be annotated '@MainActor' to avoid a Swift bug:
    //     https://github.com/pointfreeco/swift-dependencies/discussions/301
    static let liveValue = OpenURLEffect { @MainActor url in
```